src/samples/Uno.Material.Samples/Uno.Material.Samples.macOS/Uno.Material.Samples.macOS.csproj

```diff
@@ -116,15 +116,11 @@
   <ItemGroup>
     <None Include="Info.plist" />
     <None Include="Entitlements.plist" />
+    <BundleResource Include="Resources\Fonts\uno-fluentui-assets.ttf" />

```

Comment:
For macOS you need to set the "CopyToOutputDirectory" to Always

` <CopyToOutputDirectory>Always</CopyToOutputDirectory>`